src/duiker.py

```diff
@@ -380,7 +380,7 @@ def parse_args(argv):
         PS1="\\u@\\h:\\w$ "
     }}
 
-    PROMPT_COMMAND=__prompt
+    PROMPT_COMMAND='(__prompt $)'
```

Comment:
The idea here is to launch `__duiker_import` itself in a subshell, correct? What is the `$` for?